cpp/Node.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2017 Bytes Club
+ 
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+#include "Node.h"
```

Comment:
A newline at the end of the file. (Run [cpplint](https://github.com/google/styleguide/tree/gh-pages/cpplint) for more)